### PR TITLE
Update code samples

### PR DESCRIPTION
**Pull request by bidoubiwa**
See https://github.com/meilisearch/integration-guides/issues/185
fixes: #271

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -975,6 +929,161 @@ getting_started_search_md: |-
...
+  client.index("movies").updateDisplayedAttributes(displayedAttributes) { (result: Result<Task, Swift.Error>) in
+       switch result {
+       case .success(let task):
+           print(task)
+       case .failure(let error):
+           print(error)
+       }
+   }
+getting_started_communicating_with_a_protected_instance: |-
+  let client = MeiliSearch(host: "http://localhost:7700", apiKey: "master")
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:1005]:
```suggestion
  let client = MeiliSearch(host: "http://localhost:7700", apiKey: "apiKey")
```

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -304,19 +304,14 @@ update_settings_1: |-
...
+      "genres"
   ], displayedAttributes: [
       "title",
-      "description",
-      "poster",
-      "release_date",
-      "rank"
+      "overview",
+      "genres"
+      "release_date"
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:314]: I can't comment in line 322, but it should be: `distinctAttribute: "movie_id",`

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -917,27 +866,32 @@ documents_guide_add_movie_1: |-
...
       case .failure(let error):
           print(error)
       }
   }
-search_guide_2: |-
-  let parameters = SearchParameters(
-      query: "Avengers",
-      filter: "release_date > 795484800")
-  client.index("movies").search(parameters) { (result: Result<SearchResult<Movie>, Swift.Error>) in
+document_guide_add_document_primary_key: |-
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:878]: WDYT? (simplify a little) 

```swfit
  let documents: Data = """
  [
    {
      "reference_number": 287947,
      "title": "Shazam",
      "poster": "https://image.tmdb.org/t/p/w1280/xnopI5Xtky18MPhK40cZAGAOVeV.jpg",
      "overview": "A boy is given the ability to become an adult superhero in times of need with a single magic word.",
      "release_date": "2019-03-23"
    }
  ]
  """.data(using: .utf8)!

  client.index("movies").addDocuments(documents: documents, primaryKey: "reference_number") { (result: Result<Task, Swift.Error>) in
```

File: .code-samples.meilisearch.yaml [hunk 4]
```diff
@@ -975,6 +929,161 @@ getting_started_search_md: |-
...
+  client.index("movies").search { (result: Result<SearchResult<Movie>, Swift.Error>) in
+      switch result {
+      case .success(let searchResult):
+          print(searchResult)
+      case .failure(let error):
+          print(error)
+      }
+  }
+getting_started_add_meteorites: |-
+  let path = Bundle.main.url(forResource: "meteorites", withExtension: "json")
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:1015]: FYI (curiosity): `Bundle.main` is only available in apps with the SPM, but not with cocoapods.

**Review by brunoocasali — COMMENTED** [review]
I left some comments, let me know if you need anything!

File: .code-samples.meilisearch.yaml (hunk 4 above)
Comment (bidoubiwa) [reply .code-samples.meilisearch.yaml]: I have updated the way to read from a file in all examples :)

**Review by brunoocasali — DISMISSED** [review]
🔥 🦅

**Comment by bidoubiwa** [comment]
bors merge

File: .code-samples.meilisearch.yaml [hunk 5]
```diff
@@ -67,8 +67,10 @@ get_documents_1: |-
       }
   }
 add_or_replace_documents_1: |-
-  let path = Bundle.main.url(forResource: "movies", withExtension: "json")
+  let file = "movies.json"
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:70]: I think this one, is just to be a simple document addition 🤔 

image

**Review by brunoocasali — APPROVED** [review]
🦅 🦅

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**